wscript

```diff
@@ -80,6 +80,7 @@ def options(opt):
     opt.add_option('--clients', default=256, type='int', dest='clients', help='Maximum number of JACK clients')
     opt.add_option('--ports-per-application', default=2048, type='int', dest='application_ports', help='Maximum number of ports per application')
     opt.add_option('--systemd-unit', action='store_true', default=False, help='Install systemd units.')
+    opt.add_option('--example-tools', type='choice', choices=['yes', 'no'], default='yes', help='Build and install jack-example-tools. Possible values: "yes" or "no"')
```

Comment:
why the custom choice instead of the usual `action='store_true', default=True,` like other arguments already use?

Reply:
Because using a bool type option would not allow us what you communicated to me in regards to deprecation/removal in three (version) steps:
* add option to optionally build jack-example-tools (default to building the jack-example-tools)
* change option default to not build jack-example-tools
* remove option and files

A bool type option only allows to set the default (e.g. `False`) and then use it (unless there is some hidden magic in waf - would not surprise me - to automatically also have e.g. a `--not-example-tools` option or so) to set to `True` (using the action parameter).

Reply:
Okay, changed to use the autooptions stuff.